client/src/pages/Landing.tsx

```diff
@@ -1,27 +1,195 @@
-import { Link } from "react-router-dom";
+import { useEffect, useRef } from "react";
+import { Link, useNavigate } from "react-router-dom";
 import Button from "../components/Button";
+import LogoBar from "../components/LogoBar";
+// import illustrations
+import illustration1 from "../assets/illustration1.svg";
+import illustration10 from "../assets/illustration10.svg";
+import illustration19 from "../assets/illustration19.svg";
+import uploadIcon from "../assets/upload.svg";
+import editIcon from "../assets/edit.svg";
+import exportIcon from "../assets/export-icon.svg";
+
+
+export default function Landing() {
+  useEffect(() => {
+    document.title = "Lifeline | Making Deadlines Easier";
+  }, []);
+  
+  // const howItWorksSection = useRef(null);
+  
+  const navigate = useNavigate(); 
+  const routeChange = () =>{ 
+    // eslint-disable-next-line quotes
+    const path = `/upload`;; 
+    navigate(path);
+  };
 
-export default function About() {
   return (
+    // <>
+    //   <div>
```

Comment:
Tidy up by removing this un-used code block entirely. (We would always be able to find this piece of code through the git tree anytime in the future, so feel good to remove it to keep the code clean)